slither/core/solidity_types/array_type.py

```diff
@@ -32,6 +32,13 @@ def type(self):
     def length(self):
         return self._length
 
+    @property
+    def storage_size(self):
+        if self._length_value:
+            elem_size, _ = self._type.storage_size
+            return elem_size * int(self._length_value.value), True
```

Comment:
I am not sure `_length` is always defined here. It could be none for dynamic array (I am not sure).
We need to handle this case to avoid a crash